app-admin/yadm/yadm-3.1.1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..9} )
```

Comment:
what about py3.10 ?

Reply:
Will add. Just copied over from the old ebuild :D